influxdb/content.md

```diff
@@ -1,643 +1,221 @@
-
-#### INFLUXDB_DB
-
-Automatically initializes a database with the name of this environment variable.
-
-#### INFLUXDB_HTTP_AUTH_ENABLED
-
-Enables authentication. Either this must be set or `auth-enabled = true` must be set within the configuration file for any authentication-related options below to work.
+After the container starts, go to `http://localhost:8086` to access the UI.
 
-#### INFLUXDB_ADMIN_USER
+For more information, see the [InfluxDB v2 Docker documentation](https://docs.influxdata.com/influxdb/v2/install/?t=docker).
 
-The name of the admin user to be created. If this is unset, no admin user is created.
+# InfluxDB v1
 
-#### INFLUXDB_ADMIN_PASSWORD
+## How to use the InfluxDB v1 Docker image
 
-The password for the admin user configured with `INFLUXDB_ADMIN_USER`. If this is unset, a random password is generated and printed to standard out.
-
-#### INFLUXDB_USER
-
-The name of a user to be created with no privileges. If `INFLUXDB_DB` is set, this user will be granted read and write permissions for that database.
-
-#### INFLUXDB_USER_PASSWORD
-
-The password for the user configured with `INFLUXDB_USER`. If this is unset, a random password is generated and printed to standard out.
-
-#### INFLUXDB_READ_USER
-
-The name of a user to be created with read privileges on `INFLUXDB_DB`. If `INFLUXDB_DB` is not set, this user will have no granted permissions.
-
-#### INFLUXDB_READ_USER_PASSWORD
-
-The password for the user configured with `INFLUXDB_READ_USER`. If this is unset, a random password is generated and printed to standard out.
-
-#### INFLUXDB_WRITE_USER
-
-The name of a user to be created with write privileges on `INFLUXDB_DB`. If `INFLUXDB_DB` is not set, this user will have no granted permissions.
-
-#### INFLUXDB_WRITE_USER_PASSWORD
-
-The password for the user configured with `INFLUXDB_WRITE_USER`. If this is unset, a random password is generated and printed to standard out.
-
-### Initialization Files
-
-If the Docker image finds any files with the extensions `.sh` or `.iql` inside of the `/docker-entrypoint-initdb.d` folder, it will execute them. The order they are executed in is determined by the shell. This is usually alphabetical order.
-
-### Manually Initialize InfluxDB v1
-
-To manually initialize an InfluxDB v1 database, use `docker run` to call the `/init-influxdb.sh` script directly. The script takes the same initialization options as the `influxd run` command--for example:
+Use the official [InfluxDB v1 Docker image](https://hub.docker.com/_/influxdb) to start a basic instance for development or testing:
 
 ```bash
-docker run --rm \
-    -e INFLUXDB_DB=db0 \
-    -e INFLUXDB_ADMIN_USER=admin \
-    -e INFLUXDB_ADMIN_PASSWORD=supersecretpassword \
-    -e INFLUXDB_USER=telegraf -e \
-    -e INFLUXDB_USER_PASSWORD=secretpassword \
-    -v "$PWD:/var/lib/influxdb" \
-    %%IMAGE%%:1.8 /init-influxdb.sh
+docker run -d -p 8086:8086 \
+  -v $PWD:/var/lib/influxdb \
+  influxdb:1.8
 ```
```

Comment:
For v1 and v2, we should keep the examples with the startup config env variables. Most users will want those.